tools/sof-tplgreader.py

```diff
@@ -361,14 +363,15 @@ def parse_and_set_block_keyword(block_str, tplgreader):
 
     # If run general test case on WOV pipeline or ECHO REFERENCE capture pipeline, there will be error
     # due to no feed data, and they should be blocked in general test case.
-    default_block_keyword = 'kpbm:any;type:capture & echo;'
+    default_block_keyword = 'kpbm:any;kpb:any;type:capture & echo;'
     # if no filter or block item is specified, the "block_none" here will help to
     # block nothing
     block_str = "block_none"
     # user specified block items from command line
     cmd_block_str = ret_args['block'].strip() if ret_args['block'] is not None else ''
 
-    if ret_args['filter'] is not None and 'kpbm' not in ret_args['filter'] and 'echo' not in ret_args['filter']:
+    if ret_args['filter'] is not None and 'echo' not in ret_args['filter'] and \
```

Comment:
Are you sure you need  the backslash?

This is how I typically handle situations like this:

- apt install black
- cp sof-tplgreader.py for_black.py
- black for_black.py
- copy line 373 and 374 from for_black.py to sof-tplgreader.py